### PR TITLE
Bugfix: Merge BSplineSurface3D

### PR DESCRIPTION
**Pull request by sbendjebla**
* **Please check if the PR fulfills these requirements**
- [ ] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix:
- [ ] New features:
- [ ] Documentation
- [ ] Optimization

* **Other information**:

### CODE REVIEW

File: volmdlr/faces.py [hunk 1]
```diff
@@ -3540,7 +3535,8 @@ def merge_with(self, other_bspline_surface3d):
 
         if bspline_face3d.outer_contour3d.is_sharing_primitives_with(other_bspline_face3d.outer_contour3d):
 
-            xmin, xmax, ymin, ymax = self.xy_limits(other_bspline_surface3d)
+            # xmin, xmax, ymin, ymax = self.xy_limits(other_bspline_surface3d)
+            pass
```
Comment (WirajanDASILVA) [inline volmdlr/faces.py:3539]: This if statement seems to have no effect, maybe we should just delete it?

File: volmdlr/faces.py [hunk 2]
```diff
@@ -3571,34 +3567,39 @@ def merge_with(self, other_bspline_surface3d):
 
                 bsplines_new[i] = surfaces[errors.index(min(errors))]
 
-            xmin, xmax, ymin, ymax = [0] * len(bsplines_new), [1] * len(bsplines_new), [0] * \
-                len(bsplines_new), [1] * len(bsplines_new)
+            # xmin, xmax, ymin, ymax = [0] * len(bsplines_new), [1] * len(bsplines_new), [0] * \
+            #     len(bsplines_new), [1] * len(bsplines_new)
```
Comment (WirajanDASILVA) [inline volmdlr/faces.py:3571]: I would recommend deleting commented code. If you ever want to come back to what it was before, git could help you with it.

**Review by WirajanDASILVA — CHANGES_REQUESTED** [review]

File: volmdlr/faces.py (hunk 2 above)
Comment (sbendjebla) [reply volmdlr/faces.py]: Comments have been deleted !

File: volmdlr/faces.py (hunk 1 above)
Comment (sbendjebla) [reply volmdlr/faces.py:3538]: Comments have been deleted !

Comment (WirajanDASILVA) [reply volmdlr/faces.py:3538]: what about the if statement?

Comment (sbendjebla) [reply volmdlr/faces.py]: It's done!

**Review by WirajanDASILVA — APPROVED** [review]

**Pull request merged**